src/js/helpers/reviewHelper.js

```diff
@@ -343,13 +343,18 @@ export const fetchObligations = (submissionId) => {
 
 export const submissionReport = (submissionId, warning, fileType, crossType) => {
     const deferred = Q.defer();
+    const params = [];
+    if (warning !== undefined && warning !== null) {
```

Comment:
does just `if (warning)` work? That would also account for cases like passing back a warning of `''` (probably more of a concern for `fileType` but the point still stands